Makefile

```diff
@@ -4,12 +4,13 @@ export DEVELOPMENT?=false
 export AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 export AWS_REGION?=us-west-2
 export IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+export KOPS_CLUSTER_NAME=$(shell cat /dev/urandom | LC_ALL=C tr -dc '[:alpha:]' | tr '[:upper:]' '[:lower:]' | head -c 32).prod.test.pdx.kops-ci.model-rocket.aws.dev
```

Comment:
Can we use postsubmit job name to define our cluster name?

Reply:
We are generating the s3 bucket name. Wanted to follow the same logic here.

Reply:
maybe we could move this into create_store.sh and use the same number for bucket and domain name

Reply:
I don't think that is important right now though.

Reply:
I think it would be more beneficial for debugging purposes, and even for the s3 bucket name if we are generating it on every run and deleting the bucket, then we should just keep the job id in there.

Reply:
Can we just use the same S3 bucket for all clusters? We can only crate 100 per account and I'd hate to have bucket leakage prevent new clusters from getting created

Reply:
Random thought, should we just use a local path for the state store since these clusters are shortlived?  One downside of local storage is if things fail to clean up, we could get the state from s3 to do a manual clean up.  That said, we may want to add a failsafe process for clean up to avoid having to do anything manually.  Maybe like a nightly cron job to "reset" the aws account.

Reply:
kops is all set up to use s3.